### DISCUSSION TITLE
2022/05/31/Hello-Tags-Categories-LaTeX-and-Rainbow/

### DISCUSSION THREAD

**Discussion by giscus** [MrWillCom/MrWillCom.github.io#5] [Announcements]
# 2022/05/31/Hello-Tags-Categories-LaTeX-and-Rainbow/

https://blog.mrwillcom.vercel.app/2022/05/31/Hello-Tags-Categories-LaTeX-and-Rainbow/

**Comment by lujiannan** [comment]
Fantastic!

**Comment by jonie-ritie** [comment]
Dear Author,
  I cannot deploy the Waline comment onto my website, I added waline configurations to the _config.yml but doesn't work

**Comment by noskyzenx** [comment]
The Rainbow banner is so cool! Love it!
